app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -753,6 +755,17 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
                     !newCategory.isInsideRepository() -> renameCategory(oldCategory, CategoryRenameError.DestinationOutsideRepo)
                     else -> lifecycleScope.launch(Dispatchers.IO) {
                         moveFile(oldCategory.file, newCategory)
+
+                        //associate the new category with the last category's timestamp in history
+                        val preference = getSharedPreferences("recent_password_history", Context.MODE_PRIVATE)
+                        val timestamp = preference.getString(oldCategory.file.absolutePath)
```

Comment:
We should update this path to a base64 too. @msfjarvis

Reply:
Oh weird, I definitely remember going back and fixing this one. Will push another PR in a bit.